### PR TITLE
Created a user database and signup page

### PR DESCRIPTION
**Pull request by hughbric**
Co-authored-by: hughbric <mhughes36@live.co.uk>
Co-authored-by: tenzinball <tenzinball@icloud.com>
Co-authored-by: olliebrownlow <olliebrownlow@gmail.com>

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
 
 class MakersBNB < Sinatra::Base
   enable :sessions
 
   get '/' do
-    'Hello world!'
+    erb :index
+  end
+
+  post '/users' do
```
Comment (VladC24) [inline app.rb]: I believe the post method should have been to /session/signup.

File: app.rb [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
+    User.create(email: params[:email], password: [:password])
+    redirect '/spaces'
+  end
+
+  get '/spaces' do
```
Comment (VladC24) [inline app.rb]: We have created the main page /spaces/index

File: views/index.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+  <head>
+    <title>MakersBNB</title>
+  </head>
+  <body>
+    <form action='/users' method='post'>
```
Comment (VladC24) [inline views/index.erb]: action='/session/signup' maybe?

Comment (hughbric) [reply views/index.erb]: Agreed - can we plan more tomorrow regarding the sign up behaviour?

**Pull request merged**